homeassistant/components/logger.py

```diff
@@ -125,4 +130,26 @@ def async_service_handler(service):
         DOMAIN, SERVICE_SET_LEVEL, async_service_handler,
         schema=SERVICE_SET_LEVEL_SCHEMA)
 
+    @asyncio.coroutine
+    def async_dump_config(service):
+        """Dump current config to the log."""
+        logger.info("Current logger config: %s", logfilter)
+        return logfilter
+
+    hass.services.async_register(
+        DOMAIN, SERVICE_DUMP_CONFIG, async_dump_config)
+
+    @asyncio.coroutine
+    def async_list_loggers(service):
+        """Return a dictionary of available loggers with their log levels."""
+        loggers = {k: v.getEffectiveLevel()
+                   for k, v in logging.Logger.manager.loggerDict.items()
+                   if isinstance(v, logging.Logger)}
+
+        logger.info("Available loggers: %s", loggers)
```

Comment:
I don't think that this makes sense. If this is needed for the frontend, let's add an api point. Services are meant to control things, not to trigger printing some log info. It's way too much of an edge case (if a use case at all)

Reply:
Yeah, that's why I closed the documentation PR. I just left this one open to hear what kind of concerns @pvizeli would have for adding such a view/api endpoint. The end goal here would be to allow easy click-to-enable debugging on running systems, instructing users to go and click debug on inside the UI would be much easier than to forward them to use the service.

The second open question was whether the part of allowing adjusting the 'default' logger would be fine or not. Just wanted to get a clarification before putting more effort into making this happen.

Reply:
I'm fine with an API to control logging of components (we should make sure it's only limited to components). The frontend however should not talk about loggers but instead probably mention "debug mode". Bonus if it will collect the logging output of a component on the page.

Reply:
Yup, thanks for :+1: I'll create a PR soonish to adding the view as it's already done, will see what can be done for the component specific logging later on.

Reply:
Cool. So let's close this PR?